### PR TITLE
56 user must give birthday and gender to register

### PR DESCRIPTION
**Pull request by JoMuelle**
-The Register Screen needs so graphical changes (Seperate Task)

### CODE REVIEW

File: src/components/views/Registration.js [hunk 1]
```diff
@@ -81,9 +95,23 @@ const Registration = props => {
...
           />
+          <div>
+            <DatePicker
+                placeholderText="Your Birthday"
+                selected={birthday}
+                onChange={(date)=>setBirthday(date)}
+                dateFormat="dd/MM/yyyy"
+            />
+          </div>
+          <div className="registration date-picker-container">
```
Comment (daveking7) [inline src/components/views/Registration.js:106]: className missplaced

File: src/components/views/Registration.js [hunk 2]
```diff
@@ -81,9 +95,23 @@ const Registration = props => {
               value={password}
               onChange={pw => setPassword(pw)}
           />
+          <div>
```
Comment (daveking7) [inline src/components/views/Registration.js:98]: div does not have className

File: src/components/views/Registration.js [hunk 3]
```diff
@@ -32,16 +34,27 @@ FormField.propTypes = {
   onChange: PropTypes.func
 };
 
+const genderOptions = [
+    {value: 'MALE', label: 'Male'},
+    {value:  'FEMALE', label: 'Female'},
+    {value: 'OTHER', label: 'Other'}
```
Comment (daveking7) [inline src/components/views/Registration.js:40]: Do we really want Other? I see why you wouldn't want to be exclusive, but the sex will be mainly used for preferences regarding playing with people from that gender. What can one expect if he/she selects "other" in their preference setting?

**Review by daveking7 — COMMENTED** [review]
Overall good, but can you get the .scc file from the datepicker you chose, so that we can adapt it afterwards?

**Review by daveking7 — CHANGES_REQUESTED** [review]
Also, resolve Merge conflicts before merging it to Dev.

File: src/components/views/Registration.js (hunk 3 above)
Comment (serainaschraff) [reply src/components/views/Registration.js:40]: I would leave it as is.. yes we might not be able to imagine what other means but it does not require much effort to implement and we will be more inclusive..

**Comment by JoMuelle** [comment]
@daveking7 Can we close this pull request, as the "better" one with changes to the datepicker has been merged?

**Comment by daveking7** [comment]
Closing Pull Request as other has been merged.

**Pull request closed without merging**